### ISSUE TITLE
Can't import browserVersion of raml with webpack

### ISSUE THREAD

**Issue by JonathanMontane** [raml-org/raml-js-parser-2#664]
@dreamflyer said in https://github.com/raml-org/raml-js-parser-2/issues/521#issuecomment-260824554

> It because of RAML.JsonValidation && RAML. XmlValidation are external dependencies which are optional. You can simply specify in your webpack config:
> externals: [ { "RAML.JsonValidation": true, "RAML.XmlValidation": true } ]

However, even though including this line in my webpack config makes the webpack compile correctly. I get an error at runtime where I fail to load the encapsulating library because it could not find these two modules.

```
Error: Cannot find module 'RAML.JsonValidation'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at webpackUniversalModuleDefinition ...
```

What would be the steps to fix this ?

**Comment by ddenisenko** [comment]
@dreamflyer .

**Comment by ddenisenko** [comment]
@JonathanMontane

Okay, so while we are waiting for @dreamflyer, I can say, that if you wish to have those modules at runtime, you have two options.

You can either include it in HTML at runtime as described in release notes here: https://github.com/raml-org/raml-js-parser-2/releases/tag/1.1.4

Or, as an alternative, you need to include those modules in your webpack configuration, and into your package statically.

Which option do you prefer? 

Also, an important question is, which of the parser distributions you are trying to pack? There are 4 different ways the parser is distriubited: NPM with a version for Node JS, NPM with web version, Bower version and Web version archive attached to each release as zip file. So, which one do you pack?

**Comment by dreamflyer** [comment]
@JonathanMontane if i correctly understand your use case, here is working example which is showing how to package parser https://github.com/dreamflyer/raml-parser-web-usage-example. There is 2 options: with validators and without. See readme for more details. But note that second option demonstrates a little workaround of how to exclude raml-xml-validation from build, because there is the bug found which i fixed today. After fix applying you will be able to remove  line `"raml-xml-validation": xmlValidationPath` in `buildWithoutValidation.js`.

**Comment by dreamflyer** [comment]
will be published in v1.1.17.

**Comment by JonathanMontane** [comment]
@dreamflyer thanks for the example! I'll look into it.
